phpunit/class-block-library-comment-template-test.php

```diff
@@ -309,8 +307,6 @@ function test_render_block_core_comment_content_converts_to_html() {
 
 		$expected_content = "<p>Paragraph One</p>\n<p>P2L1<br />\nP2L2</p>\n<p><a href=\"https://example.com/\" rel=\"nofollow ugc\">https://example.com/</a></p>\n";
 
-		// Here we use the function prefixed with 'gutenberg_*' because it's added
-		// in the build step.
```

Comment:
I replaced `gutenberg_render_block_core_comment_template` with `$block->render()` some time ago, so this comments are not relevant anymore.